src/components/ebay-table/index.marko

```diff
@@ -0,0 +1,77 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+static var validDensity = ["compact", "relaxed"];
+$ const {
+    class: divClass,
+    density,
+    header: headers,
+    row: rows,
+    ...divInput
+} = input;
+
+<div
+    ...processHtmlAttributes(divInput)
+    class=[
+        "table",
+        density &&
+            validDensity.includes(density) &&
+            `table--density-${density}`,
+        divClass,
+    ]
+    role="group"
+    tabindex="0"
+>
+    <table>
+        <thead>
+            <tr>
+                <for|header| of=headers>
```

Comment:
I know you enforce with TS that there is at least one header, but not everybody is using that and this will error if they try to render a table without any `@header`. You should either do `headers || []` here or add a default `header: headers = []` in the destructure for `input`.

Reply:
@agliga do we have example in ebayui that if not providing required attribute it will error out? I think error out is the proper approach, just need a clean way to emit the error where developer can easily be notified.